test/integration/consul-container/test/upgrade/peering_http_test.go

```diff
@@ -372,13 +374,19 @@ func TestPeering_UpgradeToTarget_fromLatest(t *testing.T) {
 		tc.extraAssertion(appPort)
 	}
 
-	for _, oldVersion := range UpgradeFromVersions {
-		for _, tc := range tcs {
-			t.Run(fmt.Sprintf("%s upgrade from %s to %s", tc.name, oldVersion, utils.TargetVersion),
-				func(t *testing.T) {
-					run(t, tc, oldVersion, utils.TargetVersion)
-				})
+	for _, tc := range tcs {
+		//  TODO: SKIP upgrade test of peering for version 1.13 sinceAPI
+		//  changed from 1.13 to 1.14 in , PeerName to Peer in
+		//  exportConfigEntry
+		fromVersion, err := version.NewVersion(utils.LatestVersion)
+		require.NoError(t, err)
+		if fromVersion.LessThan(utils.Version_1_14) {
+			continue
```

Comment:
Might remove this, but this is also ensure tests don't fail in the future if for any reason we want to run upgrade tests for a consul version < 1.14

Reply:
I think we can remove this and put the caveats into our dev doc.